What's New: some minor edits in the enterprise section (#27532)
grafana/grafana: docs/sources/guides/whats-new-in-v7-2.md

General features are included in the Grafana Enterprise edition software.

### Report grid layout

> A new layout option is available when rendering reports: the grid layout. With this option, your report will look more similar to your dashboard by re-using the same panel layout as the one used in the Dashboard.  Learn more about the [grid layout]({{< relref "../enterprise/reporting.md#layout-and-orientation.md" >}}) in the documentation.

Reviewer: ```suggestion
A new layout option is available when rendering reports: the grid layout. With this option your report will use the panel layout from your dashboard so that what you see is what you get.  Learn more about the [grid layout]({{< relref "../enterprise/reporting.md#layout-and-orientation.md" >}}) in the documentation.
```

The grid layout is also available for the [Export dashboard as PDF]({{< relref "../enterprise/export-pdf.md" >}}) feature. 

{{< docs-imagebox img="/img/docs/enterprise/reports_grid_landscape_preview.png" max-width="500px" class="docs-image--no-shadow" >}}

### Report time range

> The time range for a report can be now customized separately for each report instead of always using the time range of the Dashboard. This means you can for example configure a report that covers the previous week or month without having to trigger the report generation when that time period ends. The default is still the time range used in the Dashboard the report is based on.

Reviewer: ```suggestion
A report can now be set up with a different time range from the dashboard it is based on. This means you no longer have to apply workarounds such as copying dashboards or carefully aligning report generation with the end of the month to generate reports that covers the period you want. The dashboard's stored time range remains the default option.
```

Author: I like it, but should it be `different time rang than the dashboard` rather than `different time range from the dashboard`? Or is this a case of "my refusal to study grammar is coming back to bite me in the ass"? :)

Reviewer: 🤔
It'd be "än" in Swedish, which is close to "than" than "from", but the English "from" is also slightly wider in meaning than the Swedish "från".

Merriam-Webster has an article on the subject where the introduction says "From is the word most of the usage guides want you to use, especially in the US, so if that’s all you wanted to know you can leave this article now, untroubled by information on semi-literate 18th-century grammarians and the mysterious mating habits of the comparative adjective." https://www.merriam-webster.com/words-at-play/different-from-or-different-than

Reviewer: The end of the article is wonderful:

> If you don’t give a fig for what nitpickers think about your language use, proceed with different than or different from depending on how you feel.
>
> If you give a fig, or part of a fig, use different from, except when beginning a clause, or when to do so would sound terrible.
>
> If you are British, or would like people to think that you spent enough time in the United Kingdom for it to have influenced your approach to language, use different to whenever you feel like it.

Author: Haha, lovely :+1:
